debian/changelog

```diff
@@ -1,3 +1,9 @@
+gnome-software (3.28.0-0endless1) unstable; urgency=medium
```

Comment:
This is not `gnome-software`.

Reply:
Sorry for the stupid mistake. I should really refrain from rushing things...